### PR TITLE
smem: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/smem.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
+> More information: <https://manned.org/smem>.
+
+- Print memory usage for current processes:
+
+`smem`
+
+- Print memory usage for current processes for a every user on a system:
```
Comment (EmilySeville7cfg) [inline pages/linux/smem.md:10]:
```suggestion
- Print memory usage for all current processes for an every user on a system:
```

File: pages/linux/smem.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
+> More information: <https://manned.org/smem>.
+
+- Print memory usage for current processes:
```
Comment (EmilySeville7cfg) [inline pages/linux/smem.md:6]:
```suggestion
- Print memory usage for all current processes:
```

It's more explicit.

File: pages/linux/smem.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`smem`
+
+- Print memory usage for current processes for a every user on a system:
+
+`smem --users`
+
+- Print memory usage for current processes for a specified user:
```
Comment (EmilySeville7cfg) [inline pages/linux/smem.md:14]:
```suggestion
- Print memory usage for all current processes for a specific user:
```

File: pages/linux/smem.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`smem --userfilter {{username}}`
```
Comment (EmilySeville7cfg) [inline pages/linux/smem.md:16]:
```suggestion
`smem --userfilter {{username}}`
```

It is not clear what regex type is supported from this code sample. What about to note about it in the description smth like (`{{regex-type}}` is used)?

File: pages/linux/smem.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# smem
+
+> Show memory usage for programs.
```
Comment (EmilySeville7cfg) [inline pages/linux/smem.md:3]:
```suggestion
> Print memory usage for programs.
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/linux/smem.md (hunk 3 above)
Comment (CleanMachine1) [reply pages/linux/smem.md:14]: From what I can tell, it is not all processes

File: pages/linux/smem.md (hunk 1 above)
Comment (mfrw) [reply pages/linux/smem.md:10]: I think adding `a/an` does not sound right. Although, not an expert but in my opinion, I guess it should be:
```suggestion
- Print memory usage for all current processes for every user on the system:
```

File: pages/linux/smem.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages/linux/smem.md:16]: Sorry.

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/linux/smem.md (hunk 4 above)
Comment (CleanMachine1) [reply pages/linux/smem.md:16]: No problem!

File: pages/linux/smem.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/smem.md:10]: @CleanMachine1 wdyt about the suggestion of mfrw?

Comment (CleanMachine1) [reply pages/linux/smem.md:10]: I dont think it is all current processes therefore don't think these changes are necessary, to be honest I don't know much about the command.

Comment (navarroaxel) [reply pages/linux/smem.md:10]: ok, let's merge then.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**